mots.csl

```diff
@@ -0,0 +1,298 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="fr-FR">
+  <!--
+
+Feuille de style CSL 
+à destination du logiciel Zotero 
+comportant les instructions de formatage des références bibliographiques et de la bibliographie de littérature secondaire définies par la revue Mots [https://journals.openedition.org/mots/76]
+réalisée par Louis Autin (louis.autin@sorbonne-universite.fr / SU)
+
+ -->
+  <!-- INFORMATIONS RELATIVES A LA FEUILLE DE STYLE -->
+  <info>
+    <title>Mots (French)</title>
+    <title-short>Mots</title-short>
+    <id>http://www.zotero.org/styles/mots</id>
+    <link href="http://www.zotero.org/styles/mots" rel="self"/>
```

Comment:
please add the style you used as a template as indicated here: https://github.com/citation-style-language/styles/blob/master/STYLE_REQUIREMENTS.md#csl-style-repository-requirements